### DISCUSSION TITLE
localhost:8086 is not showing frontend in mac

### DISCUSSION THREAD

**Discussion by Lalarukh-khan** [plausible/analytics#2592] [Self-Hosted Support] [answered]
Hi!
I finally made the project to run using cmd: `docker-compose up -d `
my docker-composer.yml file has port "8086:80". This is the code of my plausible-conf.env:
`ADMIN_USER_EMAIL='lalarukhkhan179@gmail.com'
ADMIN_USER_NAME=Lalarukh
ADMIN_USER_PWD=12345
BASE_URL=http://localhost:8086
SECRET_KEY_BASE='yHgl1OisN39p3fzScXC9sSit1dNe62FsObdQyagOm3RoaVNc8er+nK4StijCKXNTWZAcR1up/3UW4+y9f4UfrA=='`
My terminal:
![Screenshot 2023-01-13 at 3 23 47 PM](https://user-images.githubusercontent.com/72685841/212394674-5353257b-f129-43cd-b5a5-22f381a65e42.png)
- Its neither running on http://localhost:8000 nor on  http://localhost:8086

![Screenshot 2023-01-13 at 3 40 00 PM](https://user-images.githubusercontent.com/72685841/212394662-2632c672-8a71-4cc0-a818-61e7eeb9c7c9.png)

**Comment by ruslandoga** [comment]
👋 @Lalarukh-khan 

> my docker-composer.yml file has port "8086:80".

That maps port 80 in the container to port 8086 on the host. You might want these to be swapped.

**Reply by Lalarukh-khan** [reply]
@ruslandoga 
After swapping ports in docker-composer.yml file `port: 80:8086` I'm getting this bug:
`Error response from daemon: cannot stop container: 1cc447d342f6a3bbdf4decb41cf05e9a454fd6b6c906f0c6b361f3f2f9fad7a7: container 1cc447d342f6 PID 65324 is zombie and can not be killed. Use the --init option when creating containers to run an init inside the container that forwards signals and reaps processes`

**Comment by Lalarukh-khan** [comment, marked as answer]
I followed this link, have described every step in a very detail depth: [self-hosting](https://www.howtogeek.com/devops/how-to-self-host-your-website-analytics-with-plausible/)
I rewrite my ports to this: `8000:8000` in docker-composer.yml. Then rewrite this url in plausible-conf.env: `BASE_URL=http://localhost:8000`
After it I ran cmd: `docker-composer up -d`. 
Then this: `docker logs yourproject_plausible_1 --follow`
And it made to run it on the `localhost:8000`
![Screenshot 2023-01-14 at 2 02 50 PM](https://user-images.githubusercontent.com/72685841/212465341-bf1de33a-79ba-46a9-a94a-67e7624da1e4.png)
